app/views/bus_lists/show.html.haml

```diff
@@ -0,0 +1,47 @@
+= stylesheet_link_tag 'home.css'
+
+%section.section{ id: 'bus_lists_section' }
+  %h1.section-title= title "Bus List"
+  %p#notice= notice
+
+  %p Thank you for being a bus captain! Below is any information you may need about your passengers.
+
+  %ul
+    %li Only students on this list may board the bus. <strong>No exceptions</strong>- this is to ensure everyone goes through the application process & has signed RIT's waivers.
+    %li If a student was accepted but didn't RSVP or sign up for the bus, they can go to <strong>https://brickhack.io/rsvp</strong> to do so.
+    %li If any emergencies arise, call our travel director, <strong>Chris Baudouin Jr.</strong>, at <strong>(203) 873-7800</strong>.
+    %li
+      If you have any other questions, please feel free to email
+      = link_to "travel@codeRIT.org", "mailto:travel@codeRIT.org"
+      !
+
+  %p
+    %b Bus List Name:
+    = @bus_list.name
+
+  = render partial: 'bus_list_info', locals: { bus_list: @bus_list }
+
+  %p
+    %strong Passengers:
+    = "(#{@bus_list.passengers.count} total, #{@bus_list.checked_in_passengers.count} checked in)"
+
+  %table.table
+    %thead
+      %tr
+        %th First Name
+        %th Last Name
+        %th Email
+        %th Phone Number
+        %th School
+        %th Checked in?
+    %tbody
+      - @bus_list.passengers.each do |p|
+        %tr
+          %td= p.first_name
+          %td= p.last_name
+          %td= p.email
+          %td= phone_link_to p.phone
+          %td= p.school.name
+          %td= p.checked_in? ? '<span class="acc-status-accepted">Yes</span>'.html_safe : 'No'
+
+  = render partial: 'bus_list_stats', locals: { bus_list: @bus_list }
```

Comment:
This file can be deleted - it was previously moved to the hackathon manager gem

Reply:
so which files should be deleted then? since I feel like some of them been moved to the hackathon manager gem

Reply:
It looks like you already resolved the merge conflicts on the others- this is the only one that got through